include/Ark/REPL/Repl.hpp

```diff
@@ -25,6 +25,7 @@ namespace Ark
         inline void print_repl_header();
         int count_open_parentheses(const std::string& line);
         int count_open_braces(const std::string& line);
+        std::string Repl::trim_whitespace(const std::string& str);
```

Comment:
The REPL:: is not needed here, maybe it would be more efficient to return nothing and directly modify the string (taking only a reference instead of a const ref)

Reply:
Whoops. Forgot to delete that after I copied it, ha.

Yeah I'll fix that, and make it so that it directly modifies the string (I'll be able to get to it later on).

Reply:
I've made those changes.